### PR TITLE
Fix Compile Button Text Overflow Issue

### LINKED ISSUE #2256: Compile button text overflows outside of button
Screen Shot 2022-04-03 at 12 28 31 PM

### PR DESCRIPTION
**Pull request by Miladinho**
Closes #2256

### CODE REVIEW

**Comment by Miladinho** [comment]
@Aniket-Engg I wrote. you on Gitter about this, but the failing test does not seem to be caused by my modification, but rather issues on the CI server that failed to install Chrome as an intermediary step to a test

**Comment by yann300** [comment]
@Miladinho could you rollback the package-lock.json changes? you don't need that in this PR.
Thanks

**Comment by Miladinho** [comment]
@yann300 applied your requested change

File: libs/remix-ui/solidity-compiler/src/lib/css/style.css [hunk 1]
```diff
@@ -154,6 +154,11 @@
 .remixui_iconbtn {
   margin-right: 0.3em;
 }
+.remixui_btnBlock {
+  display: block;
+  width: 100%;
+  word-wrap: break-word;
```
Comment (LianaHus) [inline libs/remix-ui/solidity-compiler/src/lib/css/style.css:160]: Please use Boostrap classes when possible
w-100
text-break

Comment (Miladinho) [reply libs/remix-ui/solidity-compiler/src/lib/css/style.css:160]: sweeet, thanks for that, i just pushed the changes :)

Comment (Miladinho) [reply libs/remix-ui/solidity-compiler/src/lib/css/style.css]: @LianaHus seems like the ci pipeline failed again due to timeout, let me know if there are any other changes you need

**Comment by yann300** [comment]
thanks for the update @Miladinho 
could you rebase it please?

**Comment by Miladinho** [comment]
@yann300 done

**Review by yann300 — APPROVED** [review]

**Comment by yann300** [comment]
@Miladinho sry could you do it again, seems there are some conflicts still
thanks

**Comment by Miladinho** [comment]
@yann300 I am not sure I understand, I don't see any conflicts, it does appear though that it needs 1 more reviewer in addition to you?
Screen Shot 2022-04-06 at 3 17 20 AM

**Comment by Miladinho** [comment]
@yann300 @LianaHus seems like two reviewers required here?

**Comment by Miladinho** [comment]
@yann300 I am running an update now since 6 mins ago there was a new change in master, is that what you mean?

**Comment by yann300** [comment]
we were merging quite a lot of PRs the last two days... could you rebase again please? thanks, sorry for that.

**Pull request merged**